### PR TITLE
Prepare release 3.0.5

### PR DESCRIPTION
**Pull request by LucasK96**
Fixed publishing configuration issues

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -18,6 +18,7 @@ plugins {
     kotlin("jvm") version Versions.kotlin
     id("io.gitlab.arturbosch.detekt") version "1.17.1"
     id("org.jlleitschuh.gradle.ktlint") version "10.1.0"
+    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
```
Comment (Tetr4) [inline build.gradle.kts]: Is this required for publishing to maven central?

Comment (LucasK96) [reply build.gradle.kts]: No, i added that to try something out and forgot to remove it.

File: buildSrc/src/main/kotlin/Publishing.kt [hunk 2]
```diff
@@ -4,22 +4,26 @@ import org.gradle.api.publish.maven.MavenPublication
...
+    const val versionName = "3.0.5"
 
-    const val url = "https://github.com/trbnb/MvvmBase"
-    const val gitUrl = "https://github.com/trbnb/MvvmBase.git"
-    const val licenseUrl = "https://github.com/trbnb/MvvmBase/blob/master/LICENSE"
-    const val groupId = "de.trbnb"
+    const val url = "https://github.com/appmotion-gmbh/MvvmBase"
+    const val gitUrl = "https://github.com/appmotion-gmbh/MvvmBase.git"
+    const val licenseUrl = "https://github.com/appmotion-gmbh/MvvmBase/blob/master/LICENSE"
+    const val groupId = "io.github.appmotion-gmbh"
```
Comment (Tetr4) [inline buildSrc/src/main/kotlin/Publishing.kt]: Maybe we can also extract all this to Gradle Properties, so we remain compatible with the base repo.

**Comment by Tetr4** [comment]
LGTM 👍

**Pull request merged**